device/bgp.yaml

```diff
@@ -0,0 +1,88 @@
+components:
+  schemas:
+    Device.Bgp:
+      x-include:
+      - '../common/common.yaml#/components/schemas/Named.Object'
+      x-status: under-review
+      description: >-
+        Container of basic emulated BGP peer settings.
+
+        This container has the following optional sub containers.
+        Device.BgpAdvanced, Device.BgpCapability, Device.BgpSrTePolicy
+        
+        This container has the following optional lists of containers.
+        Device.Bgpv4RouteRange, Device.Bgpv6RouteRange
+      type: object
+      properties:
+        active:
+          description: >-
+            Activates the protocol which means the protocol can be started
```

Comment:
Activate can probably be moved to a common reference-able location like the name . 'active' if enabled means that this part of the configuration including any active 'children' nodes will be advertised to peer. If disabled, this means that though config is present , it is not taking any part of the test but can be activated at run-time to advertise just this part of the configuration to the peer.